chevah/keycert/common.py

```diff
@@ -5,11 +5,29 @@
 
 Forked from twisted.conch.ssh.common
 """
-import __builtin__
-import struct
 
+from __future__ import absolute_import, division
 from Crypto import Util
 
+import struct
+import sys
+
+_PY3 = sys.version_info > (3,)
+if _PY3:
```

Comment:
You can add the no-cover praga marker 

http://coverage.readthedocs.org/en/coverage-4.0.3/excluding.html